lib/adjective.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+//js file in lib folder
+var Adjective = function() {
+  this.happy = true;
+  this.sad = true;
+  this.hairy = true;
+  this.insane = true;
+  this.flat = true;
+  this.pathetic = true;
+  this.stinky = true;
+  this.flacid = true;
+  this.fat = true;
+  this.soft = true;
+  this.hefty = true;
+  this.sweet = true;
+};
+
+
```

Comment:
hunh...I must need to update the jscs settings. Generally, you have only one, maybe two blank lines in between lines of code.